analysers/trials/numberOfProcessInstances.py

```diff
@@ -23,10 +24,95 @@
     .set("spark.cassandra.connection.host", cassandraHost)
 sc = CassandraSparkContext(conf=conf)
 
-data = sc.cassandraTable(cassandraKeyspace, srcTable)\
-        .select("duration") \
+dataRDD = sc.cassandraTable(cassandraKeyspace, srcTable)\
+        .select("process_definition_id", "source_process_instance_id", "end_time", "start_time", "duration") \
```

Comment:
@VincenzoFerme It may actually not be necessary here, it was a leftover of a different way I was selecting data. Will fix